src/layouts/layout.jsx

```diff
@@ -150,4 +160,9 @@ function Layout({ children }) {
   );
 }
 
+Layout.propTypes = {
+  children: PropTypes.object, 
+  type: BannerType,
```

Comment:
Please remove this value

Reply:
The whole code, or which specific value?

Reply:
I was talking about the type prop, you don't use this prop, right?